### PR TITLE
===== Feature Freeze =====

### PR DESCRIPTION
**Pull request by Jordie0608**
d7aad431b333ed76038ae554c09d9974
We have too many dangerously flammable issues piled up and they're threatening to burn down the servers and @Incoming5643's CPU! So we're going be freezing the codebase to start getting rid of all those issues.

This freeze will last for a month. There don't seem to be any special details this time so here are some I made up, subject to update:
- If we manage to get to 600 issues then the firemen will leave us alone and the freeze will be over.
- Content PRs for April fools will still be accepted but only for testmerging. The PR will probably get closed, which doesn't affect testmerging, but remember not to delete your branch since that does.
- Are there prizes? Who knows; I sure don't. Hopefully we'll come up with some!

As always refactors, map and asset (sprites and sound) changes are allowed.

See #24992 for the initial discussion.

### CODE REVIEW

File: code/world.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /world
-	mob = /mob/new_player
+	mob = /mob/living/simple_animal/hostile/poison/giant_spider
```
Comment (Jalleo) [inline code/world.dm]: I approve of this change

Comment (coiax) [inline code/world.dm]: Do you want to play as a giant spider?

**Comment by calzilla1** [comment]
Are gramar fixes still counted as a fix?

**Comment by coiax** [comment]
Yes.

**Comment by Supermichael777** [comment]
I haven't even had time to waste my antag token from last time!

**Comment by PKPenguin321** [comment]
Will have an update on antag token for making a freeze fix later today

**Comment by Iamgoofball** [comment]
how does this conflict already

**Comment by Cyberboss** [comment]
@Jordie0608's fork must be old as hell, `/mob/new_player` got repathed to `/mob/dead/new_player`

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

It might be a little late for this now, but coming up with a list of bugs we consider critical and that we'd like fixed before the freeze ends might be an interesting way to do it vs 200 grammar fixes and closed dupe issues

**Comment by AnturK** [comment]
I'll give a 🌟 to anyone who fixes #22124

**Comment by jughu** [comment]
Make that three 🌟

File: code/world.dm (hunk 1 above)
Comment (MrPerson) [inline code/world.dm]: I thought we were trying to get rid of bugs?

Comment (lzimann) [inline code/world.dm]: but they are not bugs

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

Please dont use the feature frozen tag this year. It just encourages people to open features during the freeze if they know we'll reopen the PRs for them

They should at least be responsible for remaking their own PR if they are not going to follow the rules

**Comment by Tacolizard** [comment]
>take coding break over the new year feature freeze
>come back
>feature freeze

**Comment by Kromgar** [comment]
>mfw we have more issues than when we started 

at 830 right now

**Comment by calzilla1** [comment]
Under 800 woo

**Comment by calzilla1** [comment]
Are tweaks kosher?

**Comment by Tacolizard** [comment]
@calzilla1 im pretty sure they are. Though someone said balance/rebalance tweaks weren't kosher over the new year freeze

**Comment by Jalleo** [comment]
Rebalances are not allowed it says so above. 
Also we wont even get below 700 issues.

**Comment by jughu** [comment]
>Reporting bugs
>not keeping for yourself untill freeze is over

**Comment by lzimann** [comment]
\>wanting the freeze to be over

**Comment by jughu** [comment]
There is cold in my summer

**Comment by lzimann** [comment]
It's not summer here

**Comment by Kromgar** [comment]
790 wew.

Also its either spring or fall depending on your hemisphere

**Comment by Jalleo** [comment]
Have we even succeded in making the game faster for @Incoming5643?

**Comment by Jalleo** [comment]
@Incoming5643 so thats a no then.

**Comment by Kromgar** [comment]
This freeze is gonna go into fucking july isn't it

**Comment by RemieRichards** [comment]
No, because freezes last a month at maximum, and *however long until we reach the goal* as a minimum.

**Comment by Kromgar** [comment]
Next freeze at 1,000 issues

**Comment by Qbopper** [comment]
day of the melt soon

**Comment by optimumtact** [comment]
Extend ed by 1 month

**Comment by PKPenguin321** [comment]
Joke?

**Comment by optimumtact** [comment]
I don't have the authority to determine freezes

**Comment by coiax** [comment]
Is it over yet

**Comment by GunHog** [comment]
I hope not!

**Comment by coiax** [comment]
Until Aslan returns, it will always be winter and never Christmas.

**Comment by optimumtact** [comment]
It ends on the new month

**Comment by Jalleo** [comment]
I want @Jordie0608 to confirm when it ends.

**Comment by Jalleo** [comment]
Ho boy freeze over.

**Pull request closed without merging**